src/Form/Field.php

```diff
@@ -284,8 +284,9 @@ public function toArray(): array
 
         unset($array['model']);
 
-        $array['invalid']   = $this->isInvalid();
         $array['errors']    = $this->errors();
+        $array['invalid']   = $this->isInvalid();
+        $array['save']      = $this->save();
```

Comment:
To access saveable fields from fields config

Reply:
I would prefer "saveable" as array key here.